ext/snmp/snmp.c

```diff
@@ -26,6 +26,9 @@
 #include "php.h"
 #include "main/php_network.h"
 #include "ext/standard/info.h"
+#ifdef ssize_t
+# undef ssize_t
+#endif
```

Comment:
So... what's going on here again?

Reply:
php_snmp.h includes netsnmp-config.h, which has an unconditional typedef for ssize_t, but PHP already did `#define ssize_t SSIZE_T`, what results in MSVC [raising C4142](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4142?redirectedfrom=MSDN&view=vs-2019) (apparently, on x86_64 the typedef matches, but not on x86). Given that the define in the net-snmp configuration is due to a custom patch, and that newer net-snmp versions are available, this is likely better resolved by a new build, instead of fixing it here.

I switch this PR to draft, until I a new net-snmp package is available.